scss/components/_case-card.scss

```diff
@@ -35,6 +35,14 @@
     left: 20px;
     top: 15px;
   }
+
+  .civicase__case-lock {
```

Comment:
Should be `civicase__case-card__lock`. and should be defined outside of `.civicase__case-card` at the root level.